### PR TITLE
Issue 97

### PR DESCRIPTION
**Pull request by rodrigc**
Fixes for https://github.com/jmmv/kyua/issues/97

### CODE REVIEW

**Comment by rodrigc** [comment]
Fixes for https://github.com/jmmv/kyua/issues/97

File: utils/fs/operations.cpp [hunk 1]
```diff
@@ -33,6 +33,10 @@
 #endif
 
 extern "C" {
+#if defined(HAVE_SYS_PARAM_H)
```
Comment (jmmv) [inline utils/fs/operations.cpp:36]: We include sys/param.h in many places already unconditionally, so I'd remove this check.

File: testers/atf_result.c [hunk 2]
```diff
@@ -680,6 +680,8 @@ convert_result(const enum atf_status status, const int status_arg,
     }
 
     assert(false);
+    // Should never reach here
+    return kyua_error_ok();
```
Comment (jmmv) [inline testers/atf_result.c:684]: Then don't continue execution. Replace these three lines (the assertion plus your two additions) with just abort().

File: configure.ac [hunk 3]
```diff
@@ -59,7 +59,7 @@ KYUA_GETOPT
 KYUA_LAST_SIGNO
 KYUA_MEMORY
 AC_CHECK_FUNCS([putenv setenv unsetenv])
-
+AC_CHECK_HEADERS([termios.h])
```
Comment (jmmv) [inline configure.ac:62]: The two blank lines were there on purpose. Please restore the deleted one.

**Pull request closed without merging**